docs/HTTP-batchsource.md

```diff
@@ -0,0 +1,206 @@
+# HTTP Batch Source
+
+Description
+-----------
+This plugin reads data from HTTP/HTTPS pages.
+Using paginated API to retrieve data is allowed.
+Data in JSON, XML, CSV, TSV, TEXT and BLOB formats is supported.
+
+Properties
+----------
+
+**Reference Name:** Name used to uniquely identify this source for lineage, annotating metadata, etc.
+
+### General
+
+**URL:** Url to fetch to the first page.
+The url must start with a protocol (e.g. http://).
+
+Supports {pagination.index} placeholder, neccessary for pagination
```

Comment:
I think we can just leave this out here. If they want to use pagination they will read the pagination section.

Reply:
done